src/signals/incident-management/containers/IncidentDetail/components/Attachments/styles.ts

```diff
@@ -0,0 +1,136 @@
+import styled from 'styled-components'
+import {
+  themeSpacing,
+  Heading,
+  themeColor,
+  Button as AscButton,
+} from '@amsterdam/asc-ui'
+import ErrorMessage from 'components/ErrorMessage'
+import LoadingIndicator from 'components/LoadingIndicator'
+import AddNote from '../AddNote'
+import StyledUploadProgress from './UploadProgress'
+
+export const Wrapper = styled.section`
+  contain: content;
+  position: relative;
+  z-index: 0;
+`
+
+export const StyledButtonWrapper = styled.div`
+  display: flex;
+  margin-top: ${themeSpacing(2)};
+  gap: 8px;
+`
+
+export const Title = styled(Heading)`
+  margin: ${themeSpacing(4)} 0;
+`
+
+export const StyledBox = styled.div`
```

Comment:
can you use themeSpacing?

Reply:
For `margin` and `with` yes. Not for `height` and `border`